### DISCUSSION TITLE
import finplot as fplt   cause "Restarting kernel..."

### DISCUSSION THREAD

**Discussion by Olidefo** [highfestiva/finplot#406] [Q&A]
Hello,

I just found finplot and try to use it for the 1st time
I installed it on:
   - 2 different wks
   - in a cmd on a windows machine with "pip install finplot"
   - with anaconda

installation seems to be successfull... no errors !

but, when executing the command 'import finplot as fplt' the kernel crashes everytime. Onb the windows machine, in a cmd env, or in Spyder (from anaconda)

does anyone have a clear vision on that ?
thx for any help
kr
oli

**Comment by Olidefo** [comment]
![1](https://user-images.githubusercontent.com/92906242/216579592-1770a5de-9fd3-4e10-a22d-8cc6411e583d.jpg)
here is a copy of the dependencies ...

**Comment by highfestiva** [comment]
That _is_ strange, but probably a bug in Qt6, as Python generally neither has the authority or capability to cause kernel crashes. Could you check the Windows System logs (Event Log or whatever it's called)?

**Comment by Olidefo** [comment]
I tried multiple time the command "import finplot", but nothing recorded into the Event Log (system, App, ...)
Here is a copy of my screen from Spyder:
![image](https://user-images.githubusercontent.com/92906242/216723080-cf178eb0-affa-4884-a582-d6619c0fef52.png)
and here from the cmd:
![image](https://user-images.githubusercontent.com/92906242/216723283-e6bf433c-cf3f-4137-bcde-3d7f7a7d8951.png)
From the last one, you can see that 'nothing happens', no error message, but ejected from the python cmd line...

**Reply by highfestiva** [reply]
See https://github.com/highfestiva/finplot/issues/392#issuecomment-1346645674
